content/php-client/resources/PAM/asset-reference-files.md

```diff
@@ -68,6 +75,13 @@ $product = $client->getAssetReferenceFileApi()->downloadFromNotLocalizableAsset(
 file_put_contents('/tmp/bridge.jpg', $product->getContents());
 ```
 
+From v4 of the PHP client, the response is returned instead of the content. It allows to get the filename and the Mime type from the response.
```

Comment:
```suggestion
From the v4 of the PHP client, the response is returned instead of the content. It allows getting the filename and the MIME type from the response.
```